upload.go

```diff
@@ -342,12 +370,28 @@ func (ua *uploadAction) uploadError(err error) error {
 	resp, err := client.Do(req)
 
 	if err != nil {
-		return err
+		return retryAllowed, fmt.Errorf("Unable to upload error to Waldo, error: %v, url: %s", err, url)
 	}
 
+	// dumpResponse(ua.userVerbose, resp, true)
+
 	defer resp.Body.Close()
 
-	// dumpResponse(ua.userVerbose, resp, true)
+	return retryAllowed && shouldRetry(resp), ua.checkErrorStatus(resp)
+}
+
+func (ua *uploadAction) uploadErrorWithRetry(err error) error {
+	for attempts := 1; attempts <= maxNetworkAttempts; attempts++ {
+		retry, tmpErr := ua.uploadError(err, attempts < maxNetworkAttempts)
+
+		if !retry || tmpErr == nil {
+			return tmpErr
+		}
+
+		// emit(tmpErr)
+
+		// fmt.Printf("\nFailed upload error attempts: %d -- retrying…\n\n", attempts)
```

Comment:
Why the commented code ?

Reply:
Normally we want the error uploading to proceed silently (even in verbose mode). However, when debugging it is easy enough to uncomment. I didn’t feel a burning need to add a hidden “super-verbose” mode.